src/Sulu/Component/Webspace/Document/Initializer/WebspaceInitializer.php

```diff
@@ -64,9 +64,20 @@ public function __construct(
         $this->nodeManager = $nodeManager;
     }
 
+    /**
+     * {@inheritdoc}
+     */
     public function initialize(OutputInterface $output)
     {
-        foreach ($this->webspaceManager->getWebspaceCollection() as $webspace) {
+        // sort the webspaces in order that they always appear
+        // in the same order across different platforms (which
+        // helps when testing).
+        $webspaces = iterator_to_array($this->webspaceManager->getWebspaceCollection());
```

Comment:
not that bad (: but if possible it should not be there and in my opinion it should be easy to move.